std/process.d

```diff
@@ -2073,19 +2087,26 @@ auto executeShell(in char[] command,
                   const string[string] env = null,
                   Config config = Config.none,
                   size_t maxOutput = size_t.max,
-                  in char[] workDir = null)
+                  in char[] workDir = null,
+                  string shellPath = nativeShell)
     @trusted //TODO: @safe
 {
-    return executeImpl!pipeShell(command, env, config, maxOutput, workDir);
+    return executeImpl!pipeShell(command,
+                                 env,
+                                 config,
+                                 maxOutput,
+                                 workDir,
+                                 shellPath);
 }
 
 // Does the actual work for execute() and executeShell().
-private auto executeImpl(alias pipeFunc, Cmd)(
+private auto executeImpl(alias pipeFunc, Cmd, TList...)(
     Cmd commandLine,
     const string[string] env = null,
     Config config = Config.none,
     size_t maxOutput = size_t.max,
-    in char[] workDir = null)
+    in char[] workDir = null,
+    TList values = TList.init)
```

Comment:
The default argument here is meaningless AFAICS.

I think the use of variadics makes sense since it's just forwarding a number of arguments to the generic `pipeFunc`. It would probably have taken me less time to understand if it was named something proper like `ExtraArgs extraArgs` though.